Tests/Application/config/config.yml

```diff
@@ -0,0 +1,13 @@
+# Doctrine Configuration
+doctrine:
+    dbal:
```

Comment:
This should not be needed as the db should be configureed over environment variables